docs/install_guides/pypi-openvino-dev.md

```diff
@@ -3,6 +3,7 @@
 <!--- The note below is intended for master branch only for pre-release purpose. Remove it for official releases. --->
 > **NOTE**: This version is pre-release software and has not undergone full release validation or qualification. No support is offered on pre-release software and APIs/behavior are subject to change. It should NOT be incorporated into any production software/solution and instead should be used only for early testing and integration while awaiting a final release version of this software.
 
+> **NOTE**: OpenVINO™ Development Tools have been deprecated and will be discontinued with 2024.0 release.
```

Comment:
```suggestion
> **NOTE**: OpenVINO™ Development Tools package have been deprecated and will be discontinued with 2024.0 release.
```
Maybe we can also add a link to https://docs.openvino.ai/2023.1/openvino_legacy_features.html ?